docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react'
+import { Image, Button } from '@stardust-ui/react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e) => void
+}
+
+const imageStyles = {
+  width: '100%',
+}
+
+const imageButtonStyles = {
+  minWidth: '56px',
+  height: '56px',
+  padding: '0',
+  background: '#fff',
+}
+
+class GridImagePickerItem extends React.Component<GridPickerItemProps, any> {
```

Comment:
Please don't use `any` in `Component` generic, it can cause issue in the future

```suggestion
class GridImagePickerItem extends React.Component<GridPickerItemProps> {
```